ZimiKM/HW7/Task3.Included.py

```diff
@@ -0,0 +1,21 @@
+def count_match(string: str) -> dict:
```

Comment:
good job
another implementation
def count_symbols_in_row(word):
    result = {}
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result